developer-resources/frameworks-and-ides/Hardhat/hardhat.md

```diff
@@ -0,0 +1,78 @@
+# Hardhat
+
+## Introduction
+
+The VeChain SDK Hardhat Plugin seamlessly integrates Hardhat with the VeChain SDK, providing developers with a powerful toolset for smart contract development on the VeChainThor blockchain. This essential package streamlines the process of creating, testing, deploying, and interacting with smart contracts in the VeChain ecosystem.
+
+## Installation and Setup
+
+### Prerequisites
+
+Ensure Hardhat is installed in your project:
+
+``` bash
+yarn add --dev hardhat
+npx hardhat
+```
+
+### Adding VeChain Support
+
+ - Install the VeChain Hardhat plugin:
+
+    ``` bash
+    yarn add @vechain/sdk-hardhat-plugin
+    ```
+
+ - Configure your `hardhat.config.ts`:
+
+    ``` typescript
+    import '@vechain/sdk-hardhat-plugin';
+
+    import { VET_DERIVATION_PATH } from '@vechain/sdk-core';
+    import { type HttpNetworkConfig } from 'hardhat/types';
+
+    const config: HardhatUserConfig = {
+    solidity: {
+        compilers: [
+        {
+            version: '0.8.17', // Specify the first Solidity version
```

Comment:
I would add a footnote about the compiler versions supported by VeChainThor, same for the VM version